com.musala.simple.students.db/src/main/java/com/musala/simple/students/db/Database.java

```diff
@@ -0,0 +1,77 @@
+package com.musala.simple.students.db;
```

Comment:
Do the followint package changes:
 - `com.musala.simple.students.db.database` - in this new package the `Database`, `DatabaseFactory` and `DatabaseTypes` classes should be moved.
 - `com.musala.simple.students.db.database.impl` - in this new package the Mongo and other implementation classes should be moved.